pypesto/hierarchical/relative/solver.py

```diff
@@ -49,21 +109,159 @@ def solve(
             List of model output matrices, as provided in AMICI's
             ``ReturnData.y``. Same order as simulations in the
             PEtab problem.
+        ssim:
+            List of sensitivity matrices from the model, as provided in AMICI's
+            ``ReturnData.sy``. Same order as simulations in the
+            PEtab problem.
         sigma:
             List of sigma matrices from the model, as provided in AMICI's
             ``ReturnData.sigmay``. Same order as simulations in the
             PEtab problem.
-        scaled:
-            Whether to scale the results to the parameter scale specified in
-            ``problem``.
+        ssigma:
+            List of sigma sensitivity matrices from the model, as provided
+            in AMICI's ``ReturnData.ssigmay``. Same order as simulations in the
+            PEtab problem.
+        inner_parameters:
+            The computed inner parameters.
+        parameter_mapping:
+            Mapping of optimization to simulation parameters.
+        par_opt_ids:
+            Ids of outer otimization parameters.
+        par_sim_ids:
+            Ids of outer simulation parameters, includes fixed parameters.
+        snllh:
+            Empty dictionary with optimization parameters as keys.
```

Comment:
An empty dict has no keys :thinking: 

Also contradicts ` snllh: np.ndarray,`

Reply:
True, this stayed from somewhere.
changed to:
```
A vector of the same length as ``par_opt_ids`` to store the
gradients in. Will be modified in-place.
```